### ISSUE TITLE
Inconsistent codes used for agent:client and agent:server slices

### ISSUE THREAD

**Issue by jamesagnew** [IHE/ITI.BasicAudit#74]
**Section Number** 

3:5.7.3

**Issue** 

The various AuditEvent profiles listed in 3:5.7.3 all use the codes `110153 / Source Role ID` and `110152 / Destination Role ID` for the `agent:client` and `agent:server` slices respectively for the slice discriminator on `agent.type.coding`.

...that is, except for the Delete and DeletePatient profiles which use `110150 / Application` and `custodian / Custodian` instead. I'm guessing this is a copy-paste error?

References:
https://profiles.ihe.net/ITI/BALP/StructureDefinition-IHE.BasicAudit.Update.html (uses the same codes that most of the profiles use)
https://profiles.ihe.net/ITI/BALP/StructureDefinition-IHE.BasicAudit.Delete.html (uses different codes)

**Proposed Change** 

Use consistent codes.

**Priority**:
- Medium: Significant issue or clarification. Requires discussion, but should not lead to long debate.

**Comment by JohnMoehrke** [comment]
Delete is different because there is no communication of data, so source and destination don't apply. Thus I used application and custodian. so it is the difference in delete vs the other RESTful activities that move data. (I did contemplate always using App/Custodian, but these are not used in existing profiles of audit event for the CRU activities. So I stuck with what is currently used or CRU, and had to then have Delete be different).
